src/views/Reticket/Verify.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { Input, AccessoryIcon } from 'indigo-react';
+
+import View from 'components/View';
+import { useTicketInput } from 'components/Inputs';
+import { ForwardButton } from 'components/Buttons';
+
+import { useLocalRouter } from 'lib/LocalRouter';
+
+export default function Verify({ STEP_NAMES, newWallet }) {
+  const { push } = useLocalRouter();
+
+  const ticketInput = useTicketInput({
+    name: 'ticket',
+    label: 'New master ticket',
+  });
+  const ticket = ticketInput.data;
+
+  const next = () => {
+    push(STEP_NAMES.RETICKET);
+  };
+
+  const matches = newWallet.value.wallet.ticket === ticket;
+
+  const displayTicketStatus = matches ? (
+    <AccessoryIcon.Success />
+  ) : (
+    <AccessoryIcon.Failure />
+  );
+
+  return (
+    <View>
+      Prove that you downloaded the new secrets!
+      <Input {...ticketInput} accessory={displayTicketStatus} />
```

Comment:
use the same pattern as the `PassportVerify` component for adding a custom validator (which will then automatically set `accessory` for you)